src/www/diag_backup.php

```diff
@@ -496,5 +496,5 @@ function restore_config_section($section_name, $new_contents)
 include("foot.inc");
 
 if ($do_reboot) {
-    system_reboot();
```

Comment:
only one caller left, please inline function in reboot.sh, same for system_halt()